xcube/core/store/store.py

```diff
@@ -106,11 +106,11 @@ class DataStore(DataOpener, ABC):
     A data store represents a collection of data resources that can be enumerated, queried, and opened in order to
     obtain in-memory representations. A data resource may be available as different types. Therefore, many functions
     allow to specify the data type using a TypeSpecifier. A type specifier consists of a name and an arbitrary set of
-    flags, given in brackets. These flags are used to define characteristics of a type, e.g., the type specifier
-    dataset[cube] denotes a dataset which also meets the requirements of a cube. A dataset specified by
-    dataset[cube, multilevel] is a cube and has multiple levels. A type specifier with a flag is compatible to a type
-    specifer that does not have the same flag set but is otherwise similar, e.g., dataset[cube] is compatible with
-    dataset. This is not true the other way round. If no type specifier is set a default type is assumed,
+    flags, given in square brackets. These flags are used to define characteristics of a type, e.g., the type specifier
+    "dataset[cube]" denotes a dataset which also meets the requirements of a cube. A dataset specified by
+    "dataset[cube, multilevel]" is a cube and has multiple levels. A type specifier with a flag is compatible to a type
```

Comment:
```suggestion
    "dataset[cube,multilevel]" is a cube and has multiple levels. A type specifier with a flag is compatible with a type
```

storeconv.md says "without spaces".

Reply:
"Without spaces" is an additional requirement for the data accessor ids. It is not necessary for "standalone" type specifiers. Maybe this can be made clearer in the storeconv.md.